testsuite/features/init_clients/buildhost_bootstrap.feature

```diff
@@ -15,7 +15,7 @@ Feature: Bootstrap a Salt build host via the GUI
     And I check "osimage_build_host"
     And I click on "Update Activation Key"
 
-  Scenario: Bootstrap a SLES build host
+  Scenario: Bootstrap a Salt build host
```

Comment:
I would remove "Salt" word from every place of this feature. Officially we just call this component as "Build Host"

Reply:
Fixed.